asylum-connect/catalog-api
# jest adding unit testing

## Description

Adding Jest as Unit Testing, generating coverage and adding it to circle ci

- Asana ticket:

## PR Checklist



- [ ] Assign @FJKhan  as a reviewer.

## How to Test

Check Circle CI to ensure all 3 components are building:

- Build
- Unit Testing
- e2e
